### PR TITLE
Add multi-user workspace account for everyone

### PR DESCRIPTION
**Pull request by jawadqur**
Enable blanket billing account (252835817747) for everyone in heal prod and heal preprod.

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- hatchery
- metadata
- portal

File: healdata.org/manifests/hatchery/hatchery.json [hunk 1]
```diff
@@ -134,13 +247,14 @@
       "name": "(Tutorial) JCOIN Tracking Opioid Stigma Notebook",
       "image": "quay.io/cdis/jupyter-jcoin:master",
       "env": {
-        "FRAME_ANCESTORS": "https://healdata.org"
+        "FRAME_ANCESTORS": "https://preprod.healdata.org"
```
Comment (haraprasadj) [inline healdata.org/manifests/hatchery/hatchery.json:250]: some urls are pointing to preprod

**Review by jmontmaxwell — APPROVED** [review]

**Review by nmetokishlubsky — APPROVED** [review]

File: healdata.org/manifests/hatchery/hatchery.json (hunk 1 above)
Comment (jawadqur) [reply healdata.org/manifests/hatchery/hatchery.json:340]: Oof! Good catch, I copy pasted it from preprod to prod. I will fix it now!

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- hatchery
- metadata
- portal

**Review by haraprasadj — APPROVED** [review]

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- hatchery
- metadata
- portal

**Pull request merged**